devices/Ft6xx6x/samples/Program.cs

```diff
@@ -45,11 +45,66 @@ void TouchInterrupCallback(object sender, PinValueChangedEventArgs pinValueChang
     {
         var point = sensor.GetPoint(true);
         Debug.WriteLine($"ID: {point.TouchId}, X: {point.X}, Y: {point.Y}, Weight: {point.Weigth}, Misc: {point.Miscelaneous}, Event: {point.Event}");
+        DiscrimitateButtons(point.X, point.Y);
     }
     else if (points == 2)
     {
         var dp = sensor.GetDoublePoints();
         Debug.WriteLine($"ID: {dp.Point1.TouchId}, X: {dp.Point1.X}, Y: {dp.Point1.Y}, Weight: {dp.Point1.Weigth}, Misc: {dp.Point1.Miscelaneous}, Event: {dp.Point1.Event}");
         Debug.WriteLine($"ID: {dp.Point2.TouchId}, X: {dp.Point2.X}, Y: {dp.Point2.Y}, Weight: {dp.Point2.Weigth}, Misc: {dp.Point2.Miscelaneous}, Event: {dp.Point1.Event}");
     }
-}
\ No newline at end of file
+}
+
+void DiscrimitateButtons(int x, int y)
+            {  
+                const double pxPerMm = 6.90415;
+                //r in milimeters is the radious of sensitivity to detect any of the given 3 buttons
+                const double  rMm = 3.5; 
+                const int x0L = 83;
+                const int x0M = 182;
+                const int x0R = 271;
+                const int y0 = 263;
+                const String strLB = "LEFT BUTTON PRESSED";
+                const String strMB = "MIDDLE BUTTON PRESSED";
+                const String strRB = "RIGHT BUTTON PRESSED";
+                const String strXY1 = "TOUCH PANEL TOUCHED at X= ";
+                const String strXY2 = ",Y= ";
+
+                //r^2 in pixels^2
+                int r2 = (int) ((rMm * pxPerMm) *(rMm * pxPerMm)); 
+                
+                int tmpL = 0;
+                int tmpM = 0;
+                int tmpR = 0;
+                int tmpY = 0;
+
+                tmpY = (y - y0) * (y - y0);
```

Comment:
No need to change, but just noticed this: unlike what's usually considered a good practice in C, it's not necessary to declare variables upfront. So these could be in the same line as the assignment below.